models/mixpanel__daily_events.sql

```diff
@@ -127,18 +118,14 @@ final as (
         number_of_users - number_of_new_users - number_of_repeat_users as number_of_return_users,
         trailing_users_28d,
         trailing_users_7d,
-        event_type || '-' || date_day as unique_key
+        event_type || '-' || date_day as unique_key,
+        {{ mixpanel.date_today('dbt_run_date')}}
 
     from agg_event_days
 
     {% if is_incremental() %}
-
-    -- only return the most recent day of data
-    where date_day >= coalesce( (select max(date_day)  from {{ this }} ), '2010-01-01')
-
+    where date_day >= {{ mixpanel.mixpanel_lookback(from_date="max(date_day)", interval=var('lookback_window', 7), datepart='day') }}
```

Comment:
Ahh yes I will add 7 days to account for this in case a different value is set by the var.
```suggestion
    where date_day >= {{ mixpanel.mixpanel_lookback(from_date="max(date_day)", interval=var('lookback_window', 7) + 7, datepart='day') }}
```

Reply:
That's a great suggestion, this sounds great to me to account for this scenario!